scss/components/_article.scss

```diff
@@ -0,0 +1,223 @@
+                                }
+                            }
+                        }
+                        .article-text-wrap{
+                            #article-text-container{
+                                & > div:first-child{
+                                    .title-wrap{
+                                        @include md-screen{
+                                            display: flex;
+                                            align-items: center;
+                                            justify-content: center;
+                                        }
+                                        h3{
+                                            font-weight: 600;
+                                            font-size: 29px;
+                                            color: #383F43;
+                                            @include md-screen{
+                                                width: 90%;
+                                            }
+                                        }
+                                        h2{
+                                            font-weight: 600;
+                                            font-size: 35px;
+                                            color: #383F43;
+                                            @include md-screen{
+                                                width: 90%;
+                                            }
+                                        }
+                                    }
+                                    .subtext-wrap{
+                                        color: #383F43;
+                                        margin: 0;
+                                        @include md-screen{
+                                            display: flex;
+                                            align-items: center;
+                                            justify-content: center;
+                                        }
+                                        &.mb{
+                                            margin-bottom: 50px;
+                                            @include md-screen{
+                                                display: none;
+                                            }
+                                        }
+                                        p{
+                                            @include md-screen{
+                                               width: 90%;
+                                            }
+                                        }
+                                    }
+                                    .sub-heading-wrap{
+                                        border-left: 4px solid #FFCF07;
+                                        margin: 30px 0;
+                                        @include md-screen{
+                                            width: 90%;
+                                            margin: 30px auto;
```

Comment:
Why is this text getting squished 10%? It doesn't appear that way on the designs.